### DISCUSSION TITLE
AMS console app package installation issue

### DISCUSSION THREAD

**Discussion by rahul7827** [ant-media/Ant-Media-Server#5411] [General]
Hello friends

I am facing issue while package installation in console application (npm install). Installation gets stuck in the middle, I tried several times but no luck.
Having so many errors of deprecation in the module (@ngui). If anyone have already done this, suggest the way to proceed.

![image](https://github.com/ant-media/Ant-Media-Server/assets/77726801/8e4d7dbf-6756-4233-a89e-5566103c1bc4)

Thanks

**Comment by USAMAWIZARD** [comment]
its working fine on my end which npm and node version are you using 
![image](https://github.com/ant-media/Ant-Media-Server/assets/47350008/b3cfc049-a16c-4e2f-876f-d567e1480340)

**Reply by rahul78275** [reply]
Hello @USAMAWIZARD

I was using different version, It worked with the version you are using.

Thank you
